app/typer/sanity/sanity.ts

```diff
@@ -24,10 +24,12 @@ export interface ISanityDokument {
 
 export enum ESanityFlettefeltverdi {
   SØKER_NAVN = 'SØKER_NAVN',
+  INNSENDT_TID = 'INNSENDT_TID',
 }
 
 export type FlettefeltVerdier = {
   søkerNavn?: string;
+  innsendtTid?: string;
```

Comment:
Trenger vi å ha et objekt som dette som holder på alle flettefeltverdier? 

Vi kan evt. ha et enum for å holde på alle navnene og si at et flettefelt er en kombo av et navn fra enumet og verdien. Så man sender inn `flettefelt={{ flettefeltnavn: verdi fra context eller hvor det ligger }}`